card/mod/05_standard/set/abstract/attachment.rb

```diff
@@ -110,6 +110,13 @@ def upload_cache_card
   @upload_cache_card ||= Card["new_#{attachment_name}".to_sym ]
 end
 
+def cached_upload= value
+  @cached_upload = value
+end
+
+def cached_upload
+  @cached_upload
```

Comment:
I'm wondering whether the name should be more explicit about what it contains. `action_id_of_cached_upload`? If you think that's too long then at least a comment here that it is an action id would be useful